cmd/openfga/openfga.go

```diff
@@ -72,23 +72,21 @@ type svcConfig struct {
 	// OIDC authentication
 	AuthOIDCIssuer   string `default:"" split_words:"true"`
 	AuthOIDCAudience string `default:"" split_words:"true"`
+
+	// Logging
+	DevelopmentLogging bool `default:"true" split_words:"true"`
```

Comment:
Should the default be `true`?

Reply:
Yeah i think so. When people follow the `getting started` section of the readme, they want to see readable logs, and json logs aren't readable